README.md

```diff
@@ -35,6 +35,44 @@ or, if you want to start the development environment
 npm run dev
 ```
 
+## Running [rclone](https://rclone.org) against Permanent.org instances
+
+- Install rclone, using [official install guide](https://rclone.org/install/)
+
+### Setting up an rclone remote
+
+- `rclone config`
+- Select option n for new and enter a remote name such as `permanent-prod` or `permanent-dev`.
+- Set `SFTP` as storage protocol by by selecting from displayed menu, after name prompt.
+- Set an `ssh` host, the host format is `sftp.${environment}.permanent.org` for example `sftp.dev.permanent.org` for `dev` and `staging`. The prod environment is an exception and simply has host as `sftp.permanent.org`.
+- Set you permanent username once prompted for example `rlcone.joy@gmail.com`.
+- Set `SFTP` port as `22` which is the default.
+- Set `ssh` password by selecting the "Yes, type in my own password" (`y`) option, when the password options show up.
+- Set your remote password which is the password to your permanent account (in this case `rlcone.joy@gmail.com`).
+- Set every other option from this point to the end, with its default value by simply pressing enter.
+
+At the end of the configuration you should have something like this:
+
+```
+Configuration complete.
+Options:
+- type: sftp
+- host: sftp.permanent.org
+- user: nfebe@opentechstrategies.com
+- pass: *** ENCRYPTED ***
+Keep this "permanent-prod" remote?
+y) Yes this is OK (default)
+e) Edit this remote
+d) Delete this remote
+```
+If everything looks okay, you want to select okay to complete the remote setup.
```

Comment:
I'd say "y" instead of "okay" here and then make clear that the user can quit the config afterward.